app/src/vessels/components/Vessel.jsx

```diff
@@ -62,6 +63,11 @@ class Vessel extends Component {
         >
           {vessel.title}
         </div>
+        {hasWarning &&
+          <div onClick={() => this.props.showWarning(vessel.comment)}>
+            <IconButton icon="alert" />
```

Comment:
@j8seangel @satellitestudiodesign shouldn't the warning icon be displayed in the same color as the banner on top?

Reply:
@enriquetuya adding contextual colors to icons is something that I like and I'm including in the new version of the UI. Currently, the UI only uses color to tell users what things are interactive, so I believe we should keep it consistent